Makefile

```diff
@@ -353,6 +353,35 @@ kubernetes-distro-lint:
 	@echo "Linting Kubernetes distribution..."
 	$(MAKE) -C distros/kubernetes lint
 
+# Helm chart validation
+.PHONY: helm-lint
+helm-lint:
+	@echo "🎯 Validating Helm charts..."
+	@# Ensure helm is available
+	@if ! command -v helm >/dev/null 2>&1; then \
+		echo "❌ Error: helm command not found. Please install Helm first."; \
+		exit 1; \
+	fi
+	@echo "Using Helm version: $$(helm version --short)"
+	@echo ""
+	@# Main nvsentinel chart
+	@echo "Validating main nvsentinel chart..."
+	helm lint distros/kubernetes/nvsentinel/
```

Comment:
Can we also add the `helm template` command here as like we have for the individual charts?